sound/soc/sof/ipc4-mtrace.c

```diff
@@ -608,6 +608,16 @@ static void ipc4_mtrace_free(struct snd_sof_dev *sdev)
 	ipc4_mtrace_disable(sdev);
 }
 
+static int sof_ipc4_mtrace_update_pos_all_cores(struct snd_sof_dev *sdev)
```

Comment:
nitpick on the commit message: but the DSP was not able to send an IPC notification anymore.
or drop the 'a' / 'an' ?